.github/ISSUE_TEMPLATE/blog_post_ga_release.md

```diff
@@ -11,7 +11,7 @@ The information you provide here will be included in the Open Liberty GA release
 
 Please provide the following information:
 
-1. If this was previously published in a beta blog post, then provide a link to it on the next line between the `<GHA-BLOG-BETA-LINK>` tags.  If nothing has changed since the beta, you're done and can ommit the remaining steps.  If you need to make updates/alterations to the beta content, then do all the steps.  
+1. If this was previously published in a beta blog post, then provide a link to it on the next line between the `GHA-BLOG-BETA-LINK` tags.  If nothing has changed since the beta, you're done and can ommit the remaining steps.  If you need to make updates/alterations to the beta content, then do all the steps.  
```

Comment:
```suggestion
1. If this was previously published in a beta blog post, then provide a link to it on the next line between the `GHA-BLOG-BETA-LINK` tags.  If nothing has changed since the beta, you're done and can omit the remaining steps.  If you need to make updates/alterations to the beta content, then do all the steps.  
```

Reply:
The issue with regards to `<GHA-BLOG-BETA-LINK>` breaking the automation has been fixed in the automation